app/modules/intelligence/prompts/prompt_service.py

```
@@ -319,22 +324,18 @@ async def create_or_update_system_prompt(
-                .all()
-            )
-
-            return [PromptResponse.model_validate(prompt) for prompt in prompts]
-        except SQLAlchemyError as e:
-            raise PromptServiceError(
-                "Failed to get prompts by agent ID and types"
-            ) from e
+    async def get_prompts(
+    self, 
+    agent_id: str, 
+    prompt_types: List[PromptType] = [PromptType.SYSTEM], 
```

Ruff (0.8.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)